account_analytic_no_lines/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Analytic No Lines',
+    'summary': """
+        This module hides analytics lines from accounting menus and disable
```

Comment:
Can we short this summary. It's a bit long and doesn't fit good in the modules view